vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,53 @@ Upgrading Vonk
+#. AuditEvent logging: In release 3.3.0 we already added support for logging audit information to a file. With this release we add to that logging that same information in AuditEvent resources. These resources are written to the Vonk Data database (not the Administration database). Users are not allowed to update or delete these resources. See :ref:`auditing` for more background.
+#. Audit logging: we added ``[Request]`` or ``[Response]`` to the log lines so you can distinguish them better.
+#. Sort! We started implementing :ref:`sorting <restful_search_sort>`. This release provides sorting for searchparameters of the types string, number, uri, reference, datetime and token, on the repositories SQL, SQLite and Memory. On the roadmap is extending this support to MongoDb and to quantity searchparameters.
+#. :ref:`feature_terminologyintegration`: You can configure Vonk to route the terminology operations to external terminology servers. You can even configure a preferred server for certain code systems like LOINC or Snomed-CT. On the roadmap is to also allow you to use these servers for validation of codes and for token searches.
+#. We implemented `$meta-delete <http://hl7.org/fhir/R4/resource-operation-meta-delete.html>`_, see :ref:`Meta plugins <vonk_plugins_meta>`.
+#. Loading plugins can lead to unexpected errors. We made the process and the log friendlier, so you can spot configuration errors more easily:
+
+   * The log outputs the version of each of the plugins
+   * If a duplicate .dll file is found, Vonk tells you which two dlls are causing this and then exits.
+   * If you configured a plugin that you are not licensed to use, this is logged with a friendly hint to acquire a license that does allow you to use it.
+
+#. The log is now by default configured to use asynchronous logging so Vonk is not limited by the speed of the logging sinks (like the Console and the log file). Please update your logsettings.instance.json if you created your own log settings in that. See :ref:`configure_log` for more background.
+
+
+Fix
+^^^
+
+#. You could load invalid XML in the Resource.text through a JSON payload. When that resource was then retrieved in XML, it would fail with an InternalServerError. Vonk will now return an OperationOutcome telling you what the problem is. You can then correct it by using JSON.
+#. Composite searchparameters were not parsed correctly. Now they are. So you don't see warnings like ``Composite SearchParameter 'CodeSystem.context-type-quantity' doesn't have components.`` anymore.  
+#. Indexing for the _profile searchparameter was broken for R4 since Vonk 3.2.1. We fixed it. If you added new resources with Vonk 3.2.1 - 3.4.0, you need to :ref:`reindex for the Resource._profile <feature_customsp_reindex_specific>` parameter.
+#. Audit log: ``%temp%`` in the path setting was evaluated as ``<current directory>\%temp%``. Fixed that to evaluate to the systems temporary directory.
```

Comment:
Only on Windows, so: "Fixed that to evaluate to the systems temporary directory when you use a Windows system. Please change the setting to a path of your choice when using a different OS.".

Reply:
Actually it should now also work for other OS's, because I literally replace `%temp%` with the system's temporary directory.

Reply:
I tried with the latest release branch, but I get a %temp% in my current Vonk directory when I start Vonk. So on MacOs, it does not seem to work yet.

Reply:
Mmm, too bad. You may file an issue on that. Won't let it stop the release now.